packages/api/src/transifexApi.d.ts

```diff
@@ -15,21 +15,21 @@ declare class JsonApiResource {
   });
   get(key: string): any;
   set(key: string, value: any): void;
-  async reload(include: string[]): null;
-  static async get(arg:string | AnyDict): JsonApiResource;
-  async fetch(relationshipName: string, force: boolean): JsonApiResource | Collection;
-  async save(arg: AnyDict | string[]): null;
-  static async create({
+  reload(include: string[]): Promise<void>;
+  static get(arg:string | AnyDict): Promise<JsonApiResource>;
+  fetch(relationshipName: string, force: boolean): Promise<JsonApiResource | Collection>;
+  save(arg: AnyDict | string[]): Promise<void>;
+  static create({
     id: string,
     attributes: AnyDict,
     relationships: AnyDict,
     links: StringDict,
-  }): JsonApiResource;
-  async delete(): void;
-  async change(field: string, value:JsonApiResource | null): void;
-  async add(field: string, values: JsonApiResource[]): null;
-  async reset(field: string, values: JsonApiResource[]): null;
-  async remove(field: string, values: JsonApiResource[]): null;
+  }): Promise<JsonApiResource>;
+  delete(): Promise<void>;
```

Comment:
Are we removing the async because we are returning a Promise? Maybe we should keep it?

Reply:
According to TS docs, `async` is not needed in the context of the declaration file. The return value defines the type.